browser/ui/commander/brave_tab_command_source.h

```diff
@@ -3,15 +3,15 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
 
-#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_COMMANDER_TAB_COMMAND_SOURCE_H_
-#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_COMMANDER_TAB_COMMAND_SOURCE_H_
-
-#include "src/chrome/browser/ui/commander/tab_command_source.h"  // IWYU pragma: export
+#ifndef BRAVE_BROWSER_UI_COMMANDER_BRAVE_TAB_COMMAND_SOURCE_H_
+#define BRAVE_BROWSER_UI_COMMANDER_BRAVE_TAB_COMMAND_SOURCE_H_
 
 #include "chrome/browser/ui/commander/command_source.h"
 
 namespace commander {
 
+// Note: This is implemented in chromium_src because we need access to functions
+// in the anonymous namespace of TabCommandSource.
```

Comment:
Wait. Now this look wrong 😅. This comment wasn't there and it wasn't obvious why did you use `chromium_src` approach. I'd say we should keep this comment in all `CommandSource*` files and move headers to `chromium_src`. This way it will be clear why we're doing it this way.